### DISCUSSION TITLE
Help with errors on gmsh and SU2

### DISCUSSION THREAD

**Discussion by Kamranh01** [su2code/SU2#1946] [Q&A]
Hello :) wondering if I could get some help

I am doing a project involving the analysis of a rotor configuration in conditions simulating the earth and martian atmosphere at a range of altitudes within SU2

For this, I constructed a mesh containing a 2D surface mesh of the aerofoil I am using submerged within a 3D mesh of a cylindrical section of far-field, so I can apply periodic boundary conditions 

![image](https://user-images.githubusercontent.com/126915189/222822355-e8244cbb-75bb-41cc-8cdd-24f17f6c15b6.png)

All was going swimmingly until I attempted to run a simulation to failure

I keep getting the error saying there is a mismatch between the mesh and the number of points. I am aware this occurs if you save all the elements of the mesh but I avoid that, I tried specifying that I only desire physical entities to be exported and that didn't work either. I am struggling to see a solution to my problem.
![image](https://user-images.githubusercontent.com/126915189/222824022-5080c41f-b6b2-4385-8e01-48fbd7335732.png)

I have applied everything as a physical surface/volume, I am confused if I have done anything wrong

I have attached the geo file, the step file for the blade, and the config file

[gmsh.zip](https://github.com/su2code/SU2/files/10885900/gmsh.zip)

I really appreciate any help I can get, the project is beginning to drain my sanity lol

**Comment by RichRoos** [comment]
Hey @Kamranh01,

Dit you solve the problem yet? 

If not... I had the same issue some time ago when I made my own mesh for a standard, 2D airfoil case. Have you tried setting up a physical group in GMSH for the internal ''fluid''? You can specify this ''BC'' in the cfg file as MARKER_INTERNAL. I believe this fixed this error for me.

**Comment by NAnand-TUD** [comment]
Hi !

This usually happens if you export the construction points as Mesh points when exporting. While exporting if you select export only domain points (I do not recall the exact option) the mesh should work fine.

Thanks,
N Anand

**Reply by Kamranh01** [reply]
I managed to fix it, brother, the main issue is that on the extrude by rotation command within gmsh, multiple surfaces are created upon the extrusion, which is where the mismatch in points stemmed from. I reconstructed the far-field using points lines and arcs, generating my own surfaces and it works perfectly. Thanks for replying nonetheless :)
